jsdoc/Home.md

```diff
@@ -167,16 +167,24 @@ If using [Google Pay](module-braintree-web-drop-in.html#~googlePayCreateOptions)
 
 The `style-src` directive is required so that the styles for the Google Pay button can be generated by the Google Pay SDK. You may omit this directive, so long as you include style rules for the Google Pay button to satisfy [Google's brand guidelines](https://developers.google.com/pay/api/web/guides/brand-guidelines#payment-buttons).
 
-### 3D Secure Specific Directives
+### 3D Secure 1 Specific Directives
 
-If using [3D Secure](module-braintree-web-drop-in.html#~threeDSecureOptions), include these additional directives:
+If using the [3D Secure component](module-braintree-web_three-d-secure.html) with a 3D Secure 1.0 integration, include these additional directives:
 
 |             | Sandbox                           | Production                    |
 |-------------|-----------------------------------|-------------------------------|
 | script-src  | songbirdstag.cardinalcommerce.com | songbird.cardinalcommerce.com |
```

Comment:
IMO, we shouldn't differentiate between v1 and v2 in our docs here, especially since v2 is now the default for merchants on the backend and we [default to v2 in Drop-in](https://github.com/braintree/braintree-web-drop-in/blob/250061ecb680263266279565fa4b6fa30b451215/src/lib/three-d-secure.js#L23). We should just update the `frame-src` for production to be `*` and have the note about why it is this way below (as you have it)

Reply:
Still need these changes to be made, since you can't really _not_ have a 3ds 2 integration in Drop-in.